src/app/app.component.ts

```diff
@@ -121,6 +125,16 @@ export class AppComponent {
     }
   }
 
+  /** Initialise appConfig and set dependent properties */
+  initialiseAppConfig() {
```

Comment:
nit(non-blocking)
Might be more declarative to rename to `subscribeToAppConfigChanges` or similar